tests/App/Http/Controllers/ArticleControllerTest.php

```diff
@@ -146,12 +146,12 @@ function seedArticles(int $articlesCount = 4, int $collectionsCount = 2): array
 
 it('should keep highlighted articles regardless of the sorting', function () {
     $highlightedArticles = Article::factory(3)->create([
-        'published_at' => now()->addMinutes(10),
```

Comment:
We should extract some minutes here because published means `published_at` < now and  `published_at` is not `null`